pkg/server/node.go

```diff
@@ -2108,8 +2108,16 @@ type lockedMuxStream struct {
 func (s *lockedMuxStream) SendIsThreadSafe() {}
 
 func (s *lockedMuxStream) Send(e *kvpb.MuxRangeFeedEvent) error {
+	// TODO(michael): where should this be defined and what's a reasonable threshold?
+	const slowMuxStreamLockLogThreshold = 0 * time.Minute
+	start := timeutil.Now()
 	s.sendMu.Lock()
 	defer s.sendMu.Unlock()
+	if dur := time.Since(start); dur > slowMuxStreamLockLogThreshold {
+		log.Infof(s.wrapped.Context(),
+			"slow lock acquisition on stream %d for r%d took %s",
+			e.StreamID, e.RangeID, dur)
```

Comment:
lmk if this log message can be made better

Reply:
One thing to double-check here is whether the context we are wrapping will already have the streamID and rangeID in the log context, making that information a bit redunant.

Reply:
Thanks! That's a good point. Running

```
dev test pkg/kv/kvclient/rangefeed -f TestRangeFeedStartTimeExclusive -v --ignore-cache --show-logs
```

I get a log line

```
I250515 12:48:25.122205 1203 server/node.go:2117  [T1,Vsystem,n1,rangefeed=lease,mux_n=1,gen=2] 49  slow lock acquisition on stream 1 for r7 took 41ns
```

From what I can tell, `mux_n` and `gen` are neither `streamID` nor `rangeID`. Source:
https://github.com/cockroachdb/cockroach/blob/1386486175a35e3207c38fcbd12dd0705a8d0a78/pkg/kv/kvclient/kvcoord/dist_sender_mux_rangefeed.go#L364-L379

But are there different verbosity levels that will display different tags in a context?